src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -28,7 +28,9 @@ public int CreateEmptyProject()
         {
             var thisAssembly = typeof(Program).GetTypeInfo().Assembly;
             var resources = from resourceName in thisAssembly.GetManifestResourceNames()
-                            where resourceName.ToLowerInvariant().EndsWith(".cs") || resourceName.ToLowerInvariant().EndsWith(".json")
+                            where resourceName.ToLowerInvariant().EndsWith(".cs")
+                            || resourceName.ToLowerInvariant().EndsWith(".json")
+                            || resourceName.ToLowerInvariant().EndsWith(".config")
```

Comment:
Idea for the future (doesn't have to happen now): Make sure all the template files have a prefix ("Template.[blah]") and then you can filter based on that instead of maintaining a list of extensions.